arcon_state/src/backend/handles.rs

```diff
@@ -235,6 +242,12 @@ impl<S: StateType, IK: Metakey, N: Metakey> Handle<S, IK, N> {
             panic!("State handles should be registered before activation!")
         }
 
+        #[cfg(feature = "metrics")]
```

Comment:
As the bytes metrics are for a whole backend and not individual handles. Let's move the registrations to the creation of the Backend

Reply:
ah nevermind, it is for individual handles. My bad

Reply:
Ignore the comment 😄